source/includes/_campaign_quota.md

```diff
@@ -101,6 +105,8 @@ Update a quota for a specific campaign.
 ```json
 {
   "num_respondents": 50,
+  "name":"Updated Quota Name",
+  "is_click_balanced": 1,
```

Comment:
This should be `true` instead of `0`.  `1` and `0` work but we're returning a boolean, so it may be confusing to have an int here